### DISCUSSION TITLE
Support app.settings from environment in Docker image

### DISCUSSION THREAD

**Discussion by jsommr** [PostgREST/postgrest#1838] [Q&A] [answered]
There is currently no easy way to add app.settings when using the Docker image. One has to override postgrest.conf, which seems somewhat complicated. My current workaround is to create pre-request functions that fill in the needed app.settings.

Is there a chance that something like this could be implemented:

```
PGRST_APP_SETTINGS1=key=value
PGRST_APP_SETTINGS2=key=value
PGRST_APP_SETTINGSN=key=value
```

Where N goes to 9 or some other number depending on how many app settings one assumes people could need.

**Comment by steve-chavez** [comment, marked as answer]
Hey @nerfpops,

Ah, I think Wolfgang already solved this one on https://github.com/PostgREST/postgrest/pull/1691. Have you tried the latest [nightly](https://hub.docker.com/r/postgrest/postgrest/tags?page=1&ordering=last_updated)?

You should be able to do:

```
PGRST_APP_SETTINGS_key1=value1
PGRST_APP_SETTINGS_key2=value2
```

**Comment by jsommr** [comment]
I'm using a nightly from this year so I'm sure it'll work. Thank you, @steve-chavez!
https://github.com/PostgREST/postgrest/pull/1691/commits/c59c188f2c1b34f1a42255be9d348963b670f9f8#diff-691fe38065bc3b74cad12bf5fc192b4b14139b168f4bb0663fee90ce41471cceR384
